openapi.yaml

```diff
@@ -26564,6 +27640,138 @@ components:
+          x-linode-cli-display: 4
+        subnets:
+          type: array
+          description: A list of subnets associated with the VPC.
+          items:
+            $ref: '#/components/schemas/VPCSubnet'
+        created:
+          type: string
+          format: date-time
+          readOnly: true
+          x-linode-filterable: true
+          description: The date-time of VPC creation.
+          example: '2023-07-11T00:00:00'
+        updated:
+          type: string
+          format: date-time
+          readOnly: true
+          nullable: true
+          x-linode-filterable: true
+          description: The date-time of the most recent VPC update.
+          example: '2023-09-11T00:00:00'
+    VPCSubnet:
+      type: object
+      description: An object describing a VPC Subnet.
+      properties:
+        id:
+          type: integer
+          readOnly: true
+          x-linode-filterable: true
+          description: The unique ID of the VPC Subnet.
+          example: 456
+          x-linode-cli-display: 1
+        label:
+          type: string
+          minLength: 1
+          maxLength: 64
+          x-linode-filterable: true
+          description: |
+            The VPC Subnet's label, for display purposes only.
+
+            * Must be unique among the VPC's Subnets.
+            * Must contain only ASCII letters, numbers, and hyphens (`-`).
+          example: cool-vpc-subnet
+          x-linode-cli-display: 2
+        ipv4:
+          type: string
+          format: ip
+          nullable: true
+          description: |
+            IPv4 range in CIDR canonical form.
+
+            * A VPC Subnet requires either `ipv4`, `ipv6`, or both.
+            * The range must belong to a private address space as defined in [RFC1918](https://datatracker.ietf.org/doc/html/rfc1918).
+            * Allowed prefix lengths: 1-29.
+            * The range must not overlap with 192.168.128.0/17.
+            * The range must not overlap with other Subnets on the same VPC.
+          example: 10.0.0.0/24
+          x-linode-cli-display: 3
+        ipv6:
+          type: string
+          format: ip
+          nullable: true
+          description: |
+            IPv6 range in CIDR canonical form. **Currently limited to select participants in the Beta program.**
+
+            * A VPC Subnet requires either `ipv4`, `ipv6`, or both.
+            * In requests, only the prefix length is allowed, for example: `/64`. The address range is then automatically assigned.
+            * Allowed prefix lengths: 64-125.
+            * The range must not overlap with other Subnets on the same VPC.
+          example: fd04:495a:691c:971c:0000:0000:0000:0000/64
+          x-linode-cli-display: 4
+        linodes:
+          type: array
+          description: An array of Linode IDs connected to the VPC Subnet.
+          readOnly: true
+          items:
+            type: integer
+            description: ID of a Linode connected to the VPC Subnet.
```

Comment:
Unsure if we should elaborate on this or not in these docs, but here are some details that we can consider adding: 

A Linode is shown in the list if it currently has a configuration profile with a VPC interface inside this subnet, regardless of if that configuration profile is currently booted, or if the interface was maybe added after the last boot. In these cases the Linode will not actually be in the VPC, but it will appear in this list. (The other way around can not happen as a VPC interface cannot be deleted while it is being actively used. ) 

We are actually right now in the process of slightly redesigning this "linodes" property to add some more information, but I don't think that will make it into the BETA release. So it could be that we should just leave this as it is for now. (https://docs.google.com/document/d/17ztUdch9YVKXbj30kg5BJpdDtW4tCIW9ftPdxYiN8PM/edit?usp=sharing )